lte/gateway/python/integ_tests/federated_tests/fabfile.py

```diff
@@ -257,10 +290,16 @@ def test_connectivity(timeout=10):
 
     # check FEG-cloud connectivity
     print("\n### Checking FEG-Cloud connectivity ###")
+    vagrant_setup('magma', destroy_vm=False)
+    with cd(feg_docker_integ_test_path_vagrant):
+        run('docker-compose exec magmad checkin_cli.py')
+    """
```

Comment:
Remove commented code

Reply:
i will divide that function into two so we can execute that command using FEG From Host and FEG from Vagrant